.gitignore

```diff
@@ -18,3 +18,4 @@ Puppetfile.lock
 .*.sw?
 .yardoc/
 Guardfile
+.rake_tasks~
```

Comment:
where is this file coming from? Our modulesync  config wil purge it. The template is https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.gitignore.erb

Reply:
Fixed